uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/util/commands/CreateRepository.java

```diff
@@ -36,6 +38,11 @@
     private final KetchLeaderCache leaders;
     private final boolean sslVerify;
 
+    private static final List<String> HOOK_IGNORED = Arrays.asList("system",
```

Comment:
I think preferences repository is missing here

Reply:
Hi Adriel
`preferences.git` is under `system` so it would be ignored.